tasks/packages.yml

```diff
@@ -17,8 +17,30 @@
 # limitations under the License.
 #
 
+- include: packages_ubuntu.yml
+  when: ansible_distribution == 'Ubuntu'
+
 - include: packages_redhat.yml
   when: ansible_os_family == 'RedHat'
 
-- include: packages_debian.yml
-  when: ansible_os_family == 'Debian'
+- name: Stop and disable keystone
+  service:
+    enabled: False
+    name: '{{ keystone_service }}'
+    state: stopped
+  when:
+    - not ansible_distribution == 'Ubuntu'
+    - not ansible_distribution_release == 'trusty'
+    - not ansible_virtualization_type == 'docker'
```

Comment:
I had an 2nd run idempotency check failure with Trusty running inside Docker container. I'm guessing it is caused by the fake-ness of the init.